packages/state/source/repository.ts

```diff
@@ -111,6 +111,48 @@ export class Repository implements Contracts.State.Repository {
 		}
 	}
 
+	public changesToJson(): Contracts.State.RepositoryChange {
+		const set = {};
+		const forget: string[] = [];
+
+		for (const name of this.#setAttributes) {
+			if (this.#setAttributes.has(name)) {
```

Comment:
Is this check correct? Doesn't it always evaluate to `true` ?

Reply:
No. Mainsail core can register additional attributes that are not required by transactions. In this case we can skip it. For example validatorMatrix, etc.